[2024-03-21 17:51:07,630] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_lr_score 2024-03-21T17:50:00+00:00 [queued]>
[2024-03-21 17:51:07,659] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_lr_score 2024-03-21T17:50:00+00:00 [queued]>
[2024-03-21 17:51:07,660] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-03-21 17:51:07,660] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2024-03-21 17:51:07,660] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-03-21 17:51:07,676] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): compute_lr_score> on 2024-03-21T17:50:00+00:00
[2024-03-21 17:51:07,688] {standard_task_runner.py:52} INFO - Started process 28112 to run task
[2024-03-21 17:51:07,693] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'compute_lr_score', '2024-03-21T17:50:00+00:00', '--job-id', '9964', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpzzl6ozbu', '--error-file', '/tmp/tmpiel61euz']
[2024-03-21 17:51:07,694] {standard_task_runner.py:77} INFO - Job 9964: Subtask compute_lr_score
[2024-03-21 17:51:07,761] {logging_mixin.py:104} INFO - Running <TaskInstance: ***_exam.compute_lr_score 2024-03-21T17:50:00+00:00 [running]> on host 3a3d0401ac26
[2024-03-21 17:51:07,878] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=***_exam
AIRFLOW_CTX_TASK_ID=compute_lr_score
AIRFLOW_CTX_EXECUTION_DATE=2024-03-21T17:50:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-21T17:50:00+00:00
[2024-03-21 17:51:08,240] {python.py:151} INFO - Done. Returned value was: None
[2024-03-21 17:51:08,259] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=***_exam, task_id=compute_lr_score, execution_date=20240321T175000, start_date=20240321T175107, end_date=20240321T175108
[2024-03-21 17:51:08,333] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-21 17:51:08,389] {local_task_job.py:151} INFO - Task exited with return code 0
